src/tools/android/java/com/google/devtools/build/android/AndroidResourceProcessor.java

```diff
@@ -434,7 +435,8 @@ static Path processDataBindings(
 
     ProcessXmlOptions options = new ProcessXmlOptions();
     options.setAppId(packagePath);
-    options.setResInput(inputResourcesDir.toFile());
+    // Passing absolute path is required for databinding to write relative paths in layout-info.zip
+    options.setResInput(inputResourcesDir.toFile().getAbsoluteFile());
```

Comment:
This requires changes to the databinding tools, correct? ("LayoutXmlProcessor.java" mentioned in #13639)
Let's separate this fix, because updating the databinding dependencies can take a fair bit of effort.

Reply:
Yes correct, I think the change is backwards compatible but let's fix is separately as it makes sense.